syntax = "proto3";

option cc_generic_services = true;

message TestRequest {
    string query = 1;
}

message TestResponse {
    string results = 1;
}

message PingRequest {}

message PingResponse {}

service TestService {
    rpc Echo (TestRequest) returns (TestResponse) {}

    rpc Ping (PingRequest) returns (PingResponse) {}
}